### PR TITLE
Internationalization Updates

### PR DESCRIPTION
**Pull request by mazenchami**
## Please verify the following:

- [ ] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR

This PR updates how we handle `i18n`. We now check the default from the device and display the language in either `en`, `ar` or `ko` (with `fallback` on, it'll revert to `en`).

Shoutout to @leonskim for contributing the Korean translations! 🚀 🇰🇷 

## Images

|    `en`    |    `ar`    |    `ko`    |
| ------------------------------- | ------------------------------- | ------------------------------- |
| en_image | ar_image | ko_image |

### CODE REVIEW

**Comment by frankcalise** [comment]
Looking good!!

File: docs/Internationalization.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Internationalization in Ignite Apps
+
+Ignite currently setup to have Internationalization setup in English, Arabic, and Korean! This is detected on app load and will set your app to that language.
+
+## RTL
```
Comment (silasjmatson) [inline docs/Internationalization.md:5]:
```suggestion
## Right to Left languages (RTL)
```

Wherever we use acronyms and initialisms in the docs, the first usage in the doc should spell it out.

File: docs/Internationalization.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+### Removing RTL Support
+
+To remove RTL support, follow the following steps:
+
+1. `/app/i18n/i18n.ts`
+
+- remove your RTL `.json` file from the `i18n.translations` object
+- remove lines 19-21 where we allow and force RTL on the native layer
+
+2. remove all other associated logic that uses export `isRTL` variable
```
Comment (silasjmatson) [inline docs/Internationalization.md:18]:
```suggestion
2. remove all other associated logic that uses the exported `isRTL` variable
```

File: docs/Internationalization.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Adding more languages
+
+1. add you language `.json` file to `app/i18n/`, i.e. `app/i18n/fr.json`
```
Comment (silasjmatson) [inline docs/Internationalization.md:22]:
```suggestion
1. add your language `.json` file to `app/i18n/`, i.e. `app/i18n/fr.json`
```

**Review by kateinkim — APPROVED** [review]
Nice job!

**Review by jamonholmgren — APPROVED** [review]
I think we can merge this PR and track that issue separately.

File: docs/Internationalization.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Internationalization in Ignite Apps
+
+Ignite currently setup to have Internationalization setup in English, Arabic, and Korean! This is detected on app load and will set your app to that language.
+
+## Right to Left languages (RTL)
+
+Since Ignite already comes with an RTL language, Arabic, adding any new ones would work by default.
+
+NOTE: if you move between a RTL and non-RTL language, you'll have to relaunch your app twice to get the correct localization logic.
```
[the remark this answers is not in the document]
Comment (jamonholmgren) [reply docs/Internationalization.md:9]: Lizzi asked if this happens on both platforms or just iOS?

**Comment by jamonholmgren** [comment]
Just needs @frankcalise's magic touch for the conflicts cleanup, and I think we're pretty close to ready. Oh, and the new issue & issue link.

**Comment by frankcalise** [comment]
@leonskim @mazenchami 

New string to translate from `en.json`

```
"headerRight": "Log Out",
```

**Comment by leonskim** [comment]
Updated `ko.json`.
|Login|Welcome|
|---|---|
|Simulator Screen Shot - iPhone 13 - 2022-08-18 at 13 49 32|Simulator Screen Shot - iPhone 13 - 2022-08-18 at 13 49 50|

**Comment by mazenchami** [comment]
Updated `ar.json`.
|Login|Welcome|
|---|---|
| login_scren | welcom_screen |

**Pull request merged**